docs/content/toc.md

```diff
@@ -2,103 +2,103 @@
 ---
 
 ## Getting Started
-  * [Concepts](../design/)
-  * [Quickstart](../tutorials/quickstart.html)
-  * [Loading Data](../tutorials/ingestion.html)
-    * [Loading from Files](../tutorials/tutorial-batch.html)
-    * [Loading from Streams](../tutorials/tutorial-streams.html)
-    * [Loading from Kafka](../tutorials/tutorial-kafka.html)
-  * [Clustering](../tutorials/cluster.html)
+  * [Concepts](/docs/#{DRUIDVERSION}/design/)
```

Comment:
instead on relying on the search and replace, we can use the same technique we used here https://github.com/druid-io/druid-io.github.io/blob/master/_layouts/doc_page.html#L55 based on variables defined here https://github.com/druid-io/druid-io.github.io/blob/master/_layouts/doc_page.html#L24 to extract the base URL for the docs at page build time.